content/naorlivne/drone-kubernetes-apply/index.md

```diff
@@ -0,0 +1,84 @@
+---
+date: 2019-08-27T00:00:00+00:00
+title: Kubernetes
+author: naorlivne
+tags: [ deploy, kubernetes, docker ]
+logo: kubernetes.svg
+repo: naorlivne/drone-kubernetes-apply
+image: naorlivne/drone-kubernetes-apply
+---
+
+This plugin can be used to deploy applications to a Kubernetes cluster, it will create\update the given kubernetes deployments/cron jobs/pods/etc as needed.
+
+Main difference between this plugins & other kubernetes Drone plugins is that it uses kubectl under the hood to allow multiple resources to be declared in a single YAML file & supports all resources types provided by Kubernetes. 
```

Comment:
Plugin in singular, Kubernetes with an uppercase K, and please wie and instead of &

Reply:
Fixed